docs/developers/coredev/packaging.md

```diff
@@ -36,6 +36,21 @@ Due to the staging steps and CDN synchronization delays, the conda packages can
 Check {doc}`release` for more details about the conda-forge release process and maintenance tasks.
 ```
 
+#### conda packaging split
+
+We provide three different outputs in the conda package recipe:
+
+- `napari-base`: This is the package that ships the actual source and data. The runtime
+  requirements only include the basic functionality. This package is only recommended as a
+  dependency for plugins and other projects. End users should not need this, since it mostly
+  correspond to the "library layer" of `napari`
```

Comment:
I think this was a little too "scary". I've suggested slightly softer language, below.

```suggestion
- `napari-base`: This is the package that ships the actual source and data. The runtime
  requirements only include the basic functionality. This package is recommended as a
  dependency for plugins and other projects. Most end users will prefer the `napari`
  package, below, but this one may be useful for those wanting a more minimal
  environment.
```

Reply:
Applied! Thanks!